src/main/python/main_window.py

```diff
@@ -64,6 +68,16 @@ def __init__(self):
 
         self.init_menu()
 
+        # cache for via definition files
+        self.cache_path = QStandardPaths.writableLocation(QStandardPaths.CacheLocation)
+        if not os.path.exists(self.cache_path):
+            os.makedirs(self.cache_path)
+        # check if the via defitions already exist
+        if os.path.isfile(self.cache_path+"/via_keyboards.json"):
```

Comment:
here and elsewhere, use os.path.join() instead

Reply:
fixed